Resources/Prototypes/Catalog/Fills/Crates/engineering.yml

```diff
@@ -105,3 +105,20 @@
     contents:
       - id: JetpackMini
         amount: 2
+
+- type: entity
+  id: CrateAirlockKit
+  parent: CrateGenericSteel
+  components:
+  - type: StorageFill
+    contents:
+      - id: SheetSteel
+        amount: 2
+      - id: CableApcStack
+        amount: 2
+      - id: DoorElectronics
+        amount: 12
+      - id: Screwdriver
+        amount: 1
+      - id: Wrench
+        amount: 1
```

Comment:
Seems redundant given how ubiquitous these are.

I recommend keeping crates small and just letting cargo order more if they want more rather than giving them significantly more than they may want.

Reply:
fair enough

Reply:
oh wait do you think that i should also reduce components?